addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -4,9 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
-
-## [49] - 2024-01-16
+- Update references for Expression Language Injection and HTTP Parameter Pollution (Issue 8262).
+  - Exponential Entity Expansion
```

Comment:
This still isn't right. The "new" change should be under a `### Changed` heading in the Unreleased section.

```md
### Changed
- Update references for Exponential Entity Expansion (Issue 8262).
```

Line 7 is no longer needed as it was in the changelog and released as part of v49,